### ISSUE TITLE
Support shortcut to add tag

### ISSUE THREAD

**Issue by emdoe** [redsolver/noteless#54]
Support #tag as tag in the note text.

**Comment by redsolver** [comment]
I don't think this makes sense because a hashtag with some text isn't really uncommon. Detected hashtags in notes would fill up the side bar and confuse users.

**Comment by emdoe** [comment]
> Detected hashtags in notes would fill up the side bar and confuse users.
Agree. I don't like a frustrating sidebar displaying all tags too. However I think it would be convenient to show all (or some) tags near the title (e.g., on the right side) since it could help users to locate their desired note more quickly.

**Comment by emdoe** [comment]
And I think it would be great if Noteless provides users an option to preview the note text (maybe 2 or 3 line?).
